env/wflow_wcoss_dell_p3.env

```diff
@@ -0,0 +1,5 @@
+# Python environemnt for workflow on WCOSS_dell_p3
```

Comment:
Please correct the misspelling in the comments of these "wflow" files (environemnt -> environment)